src/components/graph-view.js

```diff
@@ -929,7 +929,7 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
         },
         componentUpToDate: false,
       });
-    } else if ((canCreateEdge && canCreateEdge(node)) || draggingEdge) {
```

Comment:
"shiftKey" is the name of the parameter here (representing the shift key at the time of drag start), not the current shift key state.  draggingEdge state was introduced earlier than #283, but wasn't part of the original logic either.